packages/web-components/fast-element/src/templating/repeat.ts

```diff
@@ -19,6 +18,7 @@ import {
 import type { CaptureType, SyntheticViewTemplate, ViewTemplate } from "./template.js";
 import { HTMLView, SyntheticView } from "./view.js";
 import { normalizeBinding } from "./binding.js";
+// import { when } from "./when.js";
```

Comment:
```suggestion
```

Reply:
Oops. That was from some testing. Will fix. One sec.